book/07-git-tools/sections/interactive-staging.asc

```diff
@@ -41,7 +41,7 @@ What now> 2
 Update>>
 ----
 
-To stage the TODO and index.html files, you can type the numbers:
+Aby dodać pliki `TODO` i `index.html` do przechowalni, możesz wpisać numery: 
```

Comment:
Question: Do we have one place/dictionary for common translations like:
commit, pull request etc
Maybe worth creating one

Reply:
As I looked through the v1 translation, many terms are translated inconsistently. Such a file with Polish equivalents (and their lack) would be a big help. One can refer to some [Polish books](https://helion.pl/kategorie/programowanie/git), but as far as I looked through the table of contents, sometimes there are some unfortunate translations too. Personally, I would prefer that such a list be prepared by someone who is IT proficient and a professional translator.